css/style.css

```diff
@@ -65,7 +65,39 @@ body {
   left: 10px;
 }
 
-/*end comodities*/
+.pointer-list {
+  border-radius: 50%;
+  display: block;
+  margin: 10px;
+  list-style-position: initial;
+  list-style-type: disc;
+}
+
+.pointer-list__point {
+  border-radius: 50%;
+  padding-left: 0%;
+  margin-left: 3%;
+  margin-right: 3%;
+  display: inline-block;
+}
```

Comment:
I have a suggestion.
The point distance should be a little shorter to resemble the mockup.

Reply:
22px on mobile and 29px on desktop.

Reply:
change ok.
![image](https://user-images.githubusercontent.com/47303666/58207207-835eec00-7ce2-11e9-85bd-50c969a4555b.png)